src/index.ts

```diff
@@ -274,7 +274,7 @@ function initApp() {
     });
 }
 
-async function handleProgramArgs() {
+function getEventReplayArgs() {
```

Comment:
Change name to `getProgramArgs`.
Remember this function parses the args not only for event replay but for event export, etc.